tests/in/bits.wgsl

```diff
@@ -36,4 +36,20 @@ fn main() {
     u2 = extractBits(u2, 5u, 10u);
     u3 = extractBits(u3, 5u, 10u);
     u4 = extractBits(u4, 5u, 10u);
+    i = findLsb(i);
+    i2 = findLsb(i2);
+    i3 = findLsb(i3);
```

Comment:
why do we need to call it so many times! it's not like we are checking the result anyway, we are only interested in code generation here. So calling once should be ok.

Reply:
This is one call with each argument type. I can remove some if you'd like, but each one potentially generates different code.

Reply:
I think having something like an signed int scalar and an unsigned int vector would do (i.e. 2 cases per function).